includes/class-wc-calypso-bridge-admin-setup-wizard.php

```diff
@@ -339,6 +334,47 @@ public function get_next_step_link( $step = '' ) {
 		return add_query_arg( 'step', $keys[ $step_index + 1 ], remove_query_arg( 'activate_error' ) );
 	}
 
+	/**
+	 * Save step after continuing to next step
+	 */
+	public function save_step() {
+		// @codingStandardsIgnoreStart
+		if ( ! empty( $_POST['save_step'] ) && isset( $this->steps[ $this->step ]['handler'] ) ) {
+			WC_Calypso_Bridge::record_event( 'atomic_wc_obw_step_complete', array( 'name' => $this->step ) );
+			if ( method_exists( $this, 'pre_' . $this->steps[ $this->step ]['handler'][1] ) ) {
```

Comment:
Some more string duplication here and line 346. Just makes it a bit harder to read at first glance.